std/uuid.d

```diff
@@ -1046,7 +1046,7 @@ unittest
  */
 @trusted UUID randomUUID()
```

Comment:
Note this is still trusted, as `rndGen` is not actually safe. Of course, that's no excuse to have made it trusted to begin with, but I don't want to break it now...